packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -134,6 +134,33 @@ export class InstanceTagSet {
   }
 }
 
+/**
+ * The configuration for automatically rolling back deployments in a given Deployment Group.
+ */
+export interface IAutoRollbackConfig {
```

Comment:
This interface shouldn't have a an `I` prefix

Reply:
Removed.